template.json

```diff
@@ -685,7 +685,7 @@
       },
       "strength": null,
       "stealth": false,
-      "proficient": true
+      "proficient": null
```

Comment:
This is fine for the purposes of this PR, but I would encourage you to consider ditching the template.json file (or rather just minimizing it) in `2.3.0` since we have data models now.

Example: https://github.com/foundryvtt/crucible/blob/master/template.json

Reply:
Yes that should be fine to do for 2.3. I was holding off before due to some lingering issues with system data models in v10, but those were resolved with the 303 closing release.